### ISSUE TITLE
ARIMA(0,1,1) model for multiple time series. 

### ISSUE THREAD

**Issue by dblim** [statsmodels/statsmodels#8927]
I want to make an ARIMA(0,1,1) model for three time series, X_t, Y_t, Z_t "simultaneously". In practice what this means is that I create one big time 1-d series 
          W := X_t + 3*[np.nan]  + Y_t +  3*[np.nan] + Z_t + 3*[np.nan].
I then train a ARIMA(0,1,1) with endog = W and missing = 'drop'. However, when it comes to in-sample prediction this is not giving me the result I want. With an ARIMA(0,1,1) model, I would expect the X^_0 = 0, and similarly for Y^_0. But the issue is that while X^0 = nan, Y^0 is *not* nan, because there are predictions for the 3 nans before Y starts and hence residuals as well.

I would like to be able call predict for X_t, Y_t, Z_t  separately using the trained coefficients, as if these are time series I've never seen before and I'm just "applying" the fitted ARIMA to them. I tried saving the fitted params from an ARIMA(0,1,1) model, namely ma.L1 and sigma2 and "reconstructing" the prediction by hand, but it seems to not work. Can someone help me please?

**Comment by ChadFulton** [comment]
You can do this as follows:

```python
W = X_t + 3*[np.nan] + Y_t + 3*[np.nan] + Z_t + 3*[np.nan]
mod = sm.tsa.ARIMA(W, order=(1, 0, 0), trend='n')
res = mod.smooth([0.5, 1.0])

res_X = res.apply(X_t)
res_Y = res.apply(Y_t)
res_Z = res.apply(Z_t)

predict_X = res_X.predict()
predict_Y = res_Y.predict()
predict_Z = res_Z.predict()
```

**Comment by ChadFulton** [comment]
Closing as answered, but feel free to follow up further.

**Comment by dblim** [comment]
Hi Chad, thanks for your answer. I spent some time looking at the sarimax code base but couldn’t figure out how calling the code like that guarantees no interaction between X_t and Y_t say. Does your code do this? In addition, you seem to not pass the “missing” parameter into arima?

**Comment by ChadFulton** [comment]
When you use `res_X = res.apply(X_t)` then it uses the `params` associated with `res`, but that is the only part of `res` that is retained. So to be precise, the only interaction with Y_t (and Z_t) is through the `params`.

An alternative way to achieve the same thing (which might be more transparent), but which yields identical results, is:

```python
W = X_t + 3*[np.nan] + Y_t + 3*[np.nan] + Z_t + 3*[np.nan]
mod = sm.tsa.ARIMA(W, order=(0, 1, 1), trend='n')
res = mod.fit()

mod_X = sm.tsa.ARIMA(X_t, order=(0, 1, 1), trend='n')
res_X = mod_X.smooth(res.params)

mod_Y = sm.tsa.ARIMA(Y_t, order=(0, 1, 1), trend='n')
res_Y = mod_X.smooth(res.params)

mod_Z = sm.tsa.ARIMA(Z_t, order=(0, 1, 1), trend='n')
res_Z = mod_Z.smooth(res.params)

predict_X = res_X.predict()
predict_Y = res_Y.predict()
predict_Z = res_Z.predict()

```

**Comment by dblim** [comment]
Thanks. I guess I'm just a little confused if the model.fit() call guarantees there are no interactions between X_t and Z_t during parameter estimation. It seems you don't set missing = 'drop', which I assumedis the thing to do with time series padded with nans.

**Comment by ChadFulton** [comment]
Combining the series in the way you have done (i.e. by concatenating them with NaNs in between) will always have some  interaction between the series when you estimate the parameters. If you put enough NaNs in, the interaction will be pretty small as long as the estimated MA coefficient isn't too close to 1.

If you want to do this kind of estimation exactly, you can follow the approach I suggested in #4249. But this requires you to set up a custom model.

**Comment by dblim** [comment]
You're referring to this notebook yes: https://gist.github.com/ChadFulton/583020ea3740f91bec308212d48a12db? 

At any rate, with the "enough nans padding" approach, I guess you get an "approximate" answer: If the MA coefficient is small then after k steps it looks like \theta^k, so if \theta < 1 then this should be small for k >> 0.

**Comment by ChadFulton** [comment]
Yes that’s the notebook I had in mind, but you are right that your approach should generally give you a good approximation.
